backend/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN go-licenses csv ./backend/src/apiserver > /tmp/licenses.csv && \
   go-licenses save ./backend/src/apiserver --save_path /tmp/NOTICES
 
 # 2. Compile preloaded pipeline samples
-FROM python:3.7 as compiler
+FROM python:3.9 as compiler
```

Comment:
3.11 is the current release, 3.8 will be EOL in a year or so. Since dependency management is quite an issue for this project going to 3.9 which has been out for a while now should give more breathing room.